tests/systemd_test.go

```diff
@@ -19,29 +19,44 @@ func TestSystemdFido2(t *testing.T) {
 	for _, y := range yubikeys {
 		params = append(params, y.toQemuParams()...)
 	}
-	vm, err := buildVmInstance(t, Opts{
+
+	opts := Opts{
 		disk:       "assets/systemd-fido2.img",
 		kernelArgs: []string{"rd.luks.uuid=b12cbfef-da87-429f-ac96-7dda7232c189", "root=UUID=bb351f0d-07f2-4fe4-bc53-d6ae39fa1c23"},
 		params:     params,
-		extraFiles: "fido2-assert",
-	})
+		extraFiles: "/usr/lib/booster/libfido2_plugin.so",
+	}
+
+	vm, err := buildVmInstance(t, opts)
```

Comment:
It isn't clear to me if the libfido2 library will be in the system once the VM is built. Am I wrong to assume that it's there?

Reply:
Yes it will be in the image. Any item extraFiles will be added to the image together with its dynamic libraries dependencies.